solutions/13-Equalization-Room-9.27/14.25.viamodulo.asm

```diff
@@ -0,0 +1,25 @@
+-- HUMAN RESOURCE MACHINE PROGRAM --
```

Comment:
I found the issue!

Due to the way I was mishandling path parsing errors, a clear error wasn't being reported. I fixed it on master: https://github.com/atesgoral/hrm-solutions/commit/6b880f5f9dfbb98530b71c97f696ca9d871ba295

`14.25.viamodulo.asm` should be `14.25-viamodulo.asm`. I'll merge your PR and fix it on master.

Reply:
Here's the proper error: https://travis-ci.org/atesgoral/hrm-solutions/builds/641128691#L358

Reply:
Fix: https://github.com/atesgoral/hrm-solutions/commit/d878e4aac6aa726dd7e324284573f548c5864b90